lib/worker-pipeline.js

```diff
@@ -14,7 +14,19 @@ module.exports = async function ({ targets }) {
     let fn
     try {
       const toLoad = 'file://' + t.target
-      fn = (await realImport(toLoad)).default
+
+      if (toLoad.endsWith('.ts') || toLoad.endsWith('.cts')) {
```

Comment:
Given the complexity added to this map function by this PR, can we extract it to a utility function that can be used by both `worker-pipeline.js` and `pipeline.js`.

Reply:
Done